internal/pkg/service/buffer/storage/level/local/reader/volumes.go

```diff
@@ -16,7 +16,7 @@ type Volumes struct {
 }
 
 func OpenVolumes(ctx context.Context, logger log.Logger, clock clock.Clock, path string, opts ...Option) (*Volumes, error) {
-	v, err := volume.OpenVolumes(logger, path, func(info volumeInfo) (*Volume, error) {
+	v, err := volume.DetectVolumes(logger, path, func(info volumeInfo) (*Volume, error) {
```

Comment:
`volume.OpenVolumes` (predtym `writer.OpenVolumes`) -> `volume.DetectVolumes`:
 - Mame teda `volume.Open` a `volume.DetectVolumes`, nemalo by sa to vzajomne mylit.